lib/rubocop/cop/team.rb

```diff
@@ -9,11 +9,17 @@ module Cop
     # For performance reasons, Team will first dispatch cops & forces in two groups,
     # first the ones needed for autocorrection (if any), then the rest
     # (unless autocorrections happened).
+    # rubocop:disable Metrics/ClassLength
     class Team
       # @return [Team]
```

Comment:
By the way, this API was not marked as soft-deprecated in the documentation. Is there any basis for deprecating this API?

Reply:
It was added in https://github.com/rubocop/rubocop/pull/8088 as a workaround for breakage, and the comment seemed to suggest to me that it would be removed at a later date again.

I guess, if you prefer I can make a note for rubocop v2 or remove the comment altogether if it should stay around?

Reply:
Ah, this is fine. Thank you for the explanation.